File: Universal Windows Platform Solution/Phantom of the West/Phantom of the West/Visual Novel/Chapters/Chapter 11/IPlayGuest.cs
using Phantom_of_the_West.Visual_Novel.Serialization.Event_Flagging;

namespace Phantom_of_the_West.Visual_Novel.Chapters.Chapter_11
{
	internal class IPlayGuest : StoryChoice
	{
		internal IPlayGuest() : base(StoryChoice.GetLocalizedString("ch11IPlayGuest"), 11117)
		{
		}

		public override void Select()
		{
			PotWVN vn = PotWVN.MainVN;
			vn.EventFlags.Ch11HarassedAfterWorkout = WorkoutHarassment.PlayedAlong;
			base.Select();
		}
	}
}

File: Android Project/app/src/main/java/com/christopherraleigh/phantomofthewest/visual_novel/chapters/chapter_11/IPlayGuest.java
package com.christopherraleigh.phantomofthewest.visual_novel.chapters.chapter_11;

import com.christopherraleigh.phantomofthewest.R;
import com.christopherraleigh.phantomofthewest.visual_novel.PotWVN;
import com.christopherraleigh.phantomofthewest.visual_novel.chapters.StoryChoice;
import com.christopherraleigh.phantomofthewest.visual_novel.serialization.event_flagging.WorkoutHarassment;

/**
 * Created by christopherraleigh on 2016-05-22.
 */
public class IPlayGuest extends StoryChoice {
    public IPlayGuest() {
        super(R.string.ch11IPlayGuest, 11117);
    }

    @Override
    public void select() {
        PotWVN vn = PotWVN.getMainVN();
        vn.getEventFlags().setCh11HarassedAfterWorkout(WorkoutHarassment.PlayedAlong);
        super.select();
    }
}
